Assets/MRTK/SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -576,10 +586,19 @@ public void TryUpdateQueryBufferForLayerMask(LayerMask layerMask, Vector3 pointe
                 }
             }
 
-            public bool IsColliderValidGrabbable(Collider collider, bool ignoreCollidersNotInFOV, out NearInteractionGrabbable currentGrabbable)
+            public bool IsColliderValidGrabbable(Collider collider, bool ignoreCollidersNotInFOV, out NearInteractionGrabbable currentGrabbable, LRUCache<int, NearInteractionGrabbable> componentCache)
             {
                 // Check if the collider has a grabbable component which is valid
-                currentGrabbable = collider.GetComponent<NearInteractionGrabbable>();
+                var instanceId = collider.gameObject.GetInstanceID();
```

Comment:
Could this please use non-`var`? Just from reading this line, it's not clear if the ID is represented as a string, an int, a custom type, idk!

Reply:
Done!

Reply:
done!